guides/release/components/introducing-components.md

```diff
@@ -115,62 +96,91 @@ We can include our new component into our application by using HTML tag syntax.
-    </ul>
+</template>
+```
 
-    <p>Let me know when you've nailed down the dates!</p>
-  </section>
+We've just created our first component!
 
-  <form>
-    <label for="message">Message</label>
-    <input id="message" />
-    <button type="submit">
-      Send
-    </button>
-  </form>
-</div>
+We can include our new component into our application by importing the component at the top of our  Application route component (`application.gjs`) and invoking it in the template.
+
+```gjs {data-filename="app/templates/application.gjs" data-diff="+1,+2,-5,-6,-7,-8,-9,-10,-11,-12,-13,-14,-15,-16,-17,-18,+19"}
+import ReceivedMessage from 'my-app/components/received-message';
+
+<template>
+  <div class="messages">
+    <aside>
+      <div class="avatar is-active" title="Tomster's avatar">T</div>
+    </aside>
+    <section>
+      <h4 class="username">
+        Tomster
+        <span class="local-time">their local time is 4:56pm</span>
+      </h4>
+
+      <p>
+        Hey Zoey, have you had a chance to look at the EmberConf brainstorming
+        doc I sent you?
+      </p>
+    </section>
+    <ReceivedMessage />
+
+    <aside class="current-user">
+      <div class="avatar" title="Zoey's avatar">Z</div>
+    </aside>
+    <section>
+      <h4 class="username">Zoey</h4>
+
+      <p>Hey!</p>
+
+      <p>
+        I love the ideas! I'm really excited about where this year's EmberConf
+        is going, I'm sure it's going to be the best one yet. Some quick notes:
+      </p>
+
+      <ul>
+        <li>
+          Definitely agree that we should double the coffee budget this year (it
+          really is impressive how much we go through!)
+        </li>
+        <li>
+          A blimp would definitely make the venue very easy to find, but I think
+          it might be a bit out of our budget. Maybe we could rent some
+          spotlights instead?
+        </li>
+        <li>
+          We absolutely will need more hamster wheels, last year's line was
+          <em>way</em> too long. Will get on that now before rental season hits
+          its peak.
+        </li>
+      </ul>
+
+      <p>Let me know when you've nailed down the dates!</p>
+    </section>
+
+    <form>
+      <label for="message">Message</label>
+      <input id="message" />
+      <button type="submit">
+        Send
+      </button>
+    </form>
+  </div>
+</template>
 ```
 
-A _component_ is kind of like your own custom HTML tag. You can tell that a tag refers to an Ember component because it starts with a capital letter. Built-in HTML tags start with lowercase letters (`<div>`, `<p>`, `<table>`). Our component is called `<ReceivedMessage>`, based on its name on the file system.
+A _component_ is kind of like your own custom HTML tag. When we imported the component we gave it the name `ReceivedMessage`. In general, you should always give your components names that start with a capital letter. Built-in HTML tags start with lowercase letters (`<div>`, `<p>`, `<table>`).
```

Comment:
note that this is not required.

I can do this:
```gjs

const p = <template><em>{{yield}}</em></template>;

export const div = <template><p>{{yield}}</p></template>
```

Reply:
Let's not tell users that, though... You can shadow them but troll everybody else and your future self

Reply:
mega troll, yes. 

I would also use the "by convention" phrasing here

Reply:
meh it already says "In general, you should"